src/components/snapshot/overview/SnapshotWorkspace.tsx

```diff
@@ -55,17 +29,20 @@ function SnapshotWorkspace(props: SnapshotWorkspaceProps) {
   };
 
   return (
-    <Accordion defaultExpanded className={classes.snapshotAccordion}>
+    <Accordion defaultExpanded sx={{ width: '75%' }}>
       <AccordionSummary
         data-cy="snapshot-workspace-accordion"
-        className={classes.snapshotAccordionSummary}
+        sx={{
+          fontSize: '14px',
+          lineHeight: '22px',
+          fontWeight: '600',
+          color: (theme) => theme.palette.primary.main,
```

Comment:
```suggestion
          color: 'primary.main',
```

Reply:
The string `primary.main` isn't a css color - I'm pretty sure `theme.palette.primary.main` evaluates to a real css color.